code/modules/jobs/job_types/lp_jobs.dm

```diff
@@ -0,0 +1,195 @@
+	name = "LP Engineering Specialist"
+	jobtype = /datum/job/lp/engineer
+
+	ears = /obj/item/radio/headset/headset_com
+	id = /obj/item/card/id/lpengie
+	belt = /obj/item/pda/engineering
+	gloves = /obj/item/clothing/gloves/color/captain
+	uniform = /obj/item/clothing/under/rank/engineering/engineer/maintenance_tech/lp
+	alt_uniform = /obj/item/clothing/under/rank/engineering/engineer/maintenance_tech/skirt/lp
+	dcoat = /obj/item/clothing/suit/hooded/wintercoat/engineering
+	shoes = /obj/item/clothing/shoes/jackboots
+	head = /obj/item/clothing/head/beret/eng
+	backpack_contents = list(/obj/item/melee/classic_baton/telescopic=1)
+
+	backpack = /obj/item/storage/backpack/ert/engineer
+	satchel = /obj/item/storage/backpack/satchel/eng
+	duffelbag = /obj/item/storage/backpack/duffelbag/engineering
+	courierbag = /obj/item/storage/backpack/messenger/engi
+
+
+/datum/job/lp/medic
+	access = list()
+	minimal_access = list()
+	outfit = /datum/outfit/job/lp/medic
+
+/datum/job/lp/medic/get_access()
+	return get_lpmedic_access()
+
+/datum/outfit/job/lp/medic
+	name = "LP Medical specialist"
+	jobtype = /datum/job/lp/medic
+
+	ears = /obj/item/radio/headset/headset_com
+	id = /obj/item/card/id/lpmed
+	belt = /obj/item/pda/medical
+	gloves = /obj/item/clothing/gloves/color/latex/nitrile
+	uniform = /obj/item/clothing/under/rank/medical/paramedic/lp
+	alt_uniform = /obj/item/clothing/under/rank/medical/paramedic/skirt/lp
+	dcoat = /obj/item/clothing/suit/hooded/wintercoat/medical
+	shoes = /obj/item/clothing/shoes/sneakers/white
+	head = /obj/item/clothing/head/beret/med
+	backpack_contents = list(/obj/item/melee/classic_baton/telescopic=1)
+
+	backpack = /obj/item/storage/backpack/ert/medical
+	satchel = /obj/item/storage/backpack/satchel/med
+	duffelbag = /obj/item/storage/backpack/duffelbag/med
+	courierbag = /obj/item/storage/backpack/messenger/med
+	box = /obj/item/storage/box/survival/medical
+
+
+/datum/job/lp/commissioner
+	access = list()
+	minimal_access = list()
+	outfit = /datum/outfit/job/lp/commissioner
+
+/datum/job/lp/commissioner/get_access()
+	return get_lpcomm_access()
+
+
+/datum/outfit/job/lp/commissioner
+	name = "LP Commissioner"
+	jobtype = /datum/job/lp/commissioner
+
+	ears = /obj/item/radio/headset/headset_com
+	id = /obj/item/card/id/lpcomm
+	belt = /obj/item/pda/heads/lp/commissioner
+	gloves = /obj/item/clothing/gloves/color/black
+	uniform = /obj/item/clothing/under/rank/civilian/lawyer/bluesuit
+	alt_uniform = /obj/item/clothing/under/rank/civilian/lawyer/bluesuit/skirt
+	dcoat = /obj/item/clothing/suit/jacket
+	shoes = /obj/item/clothing/shoes/jackboots
+	head = /obj/item/clothing/head/beret/command
+	backpack_contents = list(/obj/item/melee/classic_baton/telescopic=1)
+
+	backpack = /obj/item/storage/backpack/captain
+	satchel = /obj/item/storage/backpack/satchel/cap
+	duffelbag = /obj/item/storage/backpack/duffelbag/captain
+	courierbag = /obj/item/storage/backpack/messenger/com
+
+
+/datum/job/scientist/lp
```

Comment:
why do scientist and miner have lp subtypes and all the other guys are subtypes of lp

Reply:
nvm i think i know what youre getting at

Reply:
Yeah, the comm and the power rangers are all subtypes of the Lieutenant to give them special access.
The rest of the standing crew are subtypes of their normal jobs to compliment the ship.